awesome_cart/awc.py

```diff
@@ -1652,6 +1677,133 @@ def cart(data=None, action=None):
 			"message": "Unknown Command."
 		}, awc_session, quotation)
 
+def apply_coupon(awc, awc_session, quotation, coupon, quotation_is_dirty):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [apply_coupon is too complex (17)](https://app.codacy.com/manual/DigiThinkIT/awesome_cart/pullRequest?prid=4737459)